platform/security/pdp/security-pdp-xacmlrealm/pom.xml

```diff
@@ -75,6 +95,13 @@
                 <artifactId>maven-bundle-plugin</artifactId>
                 <configuration>
                     <instructions>
+                        <Embed-Dependency>
+                            guava,
+                            ogc-tools-gml-jts,
+                            catalog-transformer-xml-binding,
```

Comment:
Did you really need to add all of these embeds? I'm confused why you'd have needed any but guava. In fact, I don't see where these dependencies are declared in your pom.